assets/jsons/translations/en.json

```diff
@@ -279,6 +279,18 @@
                     "error-notification": {
                         "message": "An error occured, unable to change symlinks settings."
                     }
+                },
+                "use-system-proxy": {
+                    "title": "Use system proxy",
+                    "description": "BSMansger will make network requests through system proxy.",
+                    "modal": {
+                        "title": "Restart Needed",
+                        "body": "Turning off system proxy will quit and re-launch BSManager. Are you sure you want to do this?",
+                        "confirm-btn": "Yes I'm sure"
+                    },
+                    "error-notification": {
+                        "message": "An error occured, unable to use system proxy."
+                    }
```

Comment:
Some update to the text
<pre>
title: Use System Proxy
description: BSManager will make network requests through your system proxy.
message: An error occurred, unable to change system proxy settings.
</pre>

Also check if you could do translation for other language, we use gpt-4 for the translation btw.

---

Also as a favor, `occurred` is misspelled on the other settings as well, can you fix them in your next commit, thanks thanks.

Reply:
Done.

Reply:
Done.